EasyPost.Tests/AddressTest.cs

```diff
@@ -39,54 +63,95 @@ public class AddressTest {
                 { "country", "US" },
                 { "zip", "94107" }
             };
-            Address address = Address.Create(parameters);
+            var address = Address.CreateAndVerify(parameters);
             Assert.IsNotNull(address.id);
-            Assert.AreEqual(address.company, "Simpler Postage Inc");
+            Assert.AreEqual(address.company, "SIMPLER POSTAGE INC");
             Assert.IsNull(address.name);
+            Assert.AreEqual(address.verifications.delivery.success, true);
+        }
 
-            Address retrieved = Address.Retrieve(address.id);
-            Assert.AreEqual(address.id, retrieved.id);
+        [TestMethod]
+        public void TestCreateInstance()
+        {
+            _address.Create();
+            Assert.IsNotNull(_address.id);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(HttpException))]
+        public void TestCreateWithStrictVerifications()
+        {
+            var parameters = new Dictionary<string, object>
+            {
+                { "company", "Simpler Postage Inc" },
+                { "street1", "123 Fake Street" },
+                { "zip", "94107" },
+                { "strict_verifications", new List<string> { "delivery", "zip4" } }
+            };
+
+            var address = Address.Create(parameters);
         }
 
         [TestMethod]
-        public void TestCreateWithVerifications() {
-            Dictionary<string, object> parameters = new Dictionary<string, object>() {
+        public void TestCreateWithVerifications()
+        {
+            var parameters = new Dictionary<string, object>
+            {
                 { "company", "Simpler Postage Inc" },
                 { "street1", "164 Townsend Street" },
                 { "street2", "Unit 1" },
                 { "city", "San Francisco" },
                 { "state", "CA" },
                 { "country", "US" },
                 { "zip", "94107" },
-                { "verifications", new List<string>() { "delivery", "zip4" } }
+                { "verifications", new List<string> { "delivery", "zip4" } }
             };
 
-            Address address = Address.Create(parameters);
+            var address = Address.Create(parameters);
```

Comment:
In C#, you can use `var` rather than having to specifically declare the type for most variables, making it more Pythonic rather than Java-ic.

It's mostly just a dev choice. I personally like to use it, coming from Python, because it cuts down on clutter and gets out of the way of my casts. I think it makes it easier to write without having to type out, i.e. `List<Dictionary<string, object>>` every time I want to make a new list of dictionaries.

There's a [bunch of arguments](https://intellitect.com/when-to-use-and-not-use-var-in-c/) for and against it, though. We can pick a side if we want.

Reply:
There is no performance downgrade if we use `var`, but using the explicit type can improve the readability in some situations.

articles:  
https://docs.microsoft.com/en-us/visualstudio/ide/reference/convert-var-to-explicit-type?view=vs-2022
https://www.adrienvandriel.com/articles/what-is-the-var-keyword-in-csharp-and-best-practices#:~:text=Performance%20impact%20of%20using%20VAR,have%20some%20performance%20implications%20though.

Reply:
In fact, turns out that `dotnet`'s .editorconfig enforces explicit types, and I had accidentally overridden the setting.
e352275 rectifies this.
So, for now, we'll go with explicit declaration per `dotnet`'s style.

Reply:
So for clarity, we are putting these back to the type they are instead of using var? I like the idea of using var, though with a statically typed language like C# it may be nice to retain the typing as much as possible, though I'm with you that `var` is nice ha.